File: src/main/java/problems/Uri1073_QuadradoDePares.java
package problems;

import java.io.IOException;
import java.util.Scanner;

public class Uri1073_QuadradoDePares {

    public static void main(String[] args) throws IOException {
        Scanner ler = new Scanner(System.in);

        int n = ler.nextInt();

        for (int i = 2; i <= n; i += 2) {
            System.out.println(i + "^2 = " + (i * i));
        }
    }
}

File: src/main/kotlin/problems/Uri1073_QuadradoDePares.kt
package problems

import java.util.Scanner

fun main(args: Array<String>) {
    val ler = Scanner(System.`in`)
    val numero = ler.nextInt()
    for (numeroAtual in 2..numero step 2)
        println("$numeroAtual^2 = ${numeroAtual * numeroAtual}")
}

/*
URI Online Judge | 1073
Quadrado de Pares
Adaptado por Neilor Tonin, URI  Brasil

Timelimit: 1
Leia um valor inteiro N. Apresente o quadrado de cada um dos valores pares, de 1 até N, inclusive N, se for o caso.

Entrada
A entrada contém um valor inteiro N (5 < N < 2000).

Saída
Imprima o quadrado de cada um dos valores pares, de 1 até N, conforme o exemplo abaixo.

Tome cuidado! Algumas linguagens tem por padrão apresentarem como saída 1e+006 ao invés de 1000000 o que ocasionará resposta errada. Neste caso, configure a precisão adequadamente para que isso não ocorra.

Exemplo de Entrada	Exemplo de Saída
6

2^2 = 4
4^2 = 16
6^2 = 36
 */